test/test_main.py

```diff
@@ -44,17 +55,29 @@ def test_invocation(filename):
     print()
     print(f"{len(results)} errors")
 
-    if gherkin_results:
+    def print_tabulate(gherkin_results):
         print(tabulate.tabulate(
             [[c or '' for c in r] for r in gherkin_results],
             maxcolwidths=[30] * len(gherkin_results[0]),
             tablefmt="simple_grid"
         ))
+    
+    if gherkin_results and not base.startswith("pass-"):
+        print_tabulate(gherkin_results)
```

Comment:
I wouldn't do this `... and not base.startswith("pass-")`. If we're developing new features and something that we thinks should pass raises an error, it's nice to see that in the console output.

Reply:
[We still do](https://github.com/buildingSMART/ifc-gherkin-rules/blob/a787445de8fa18c26a57fa60343a1360edb2775b/test/test_main.py#L80) .In case of a pass testfile, the gherkin_result list is modified to describe that the found error is actually a passed result and print it out afterwards with tabulate.
For example, here we see that a single pass file activates the rule through three scenarios
![image](https://github.com/user-attachments/assets/3214e41b-f204-45e1-8dac-dfe3c31e3edb)

Reply:
Thanks for clarifying. I think we should figure out another way though to pass the activation state than through raising an exception. That probably will affect this bit as well.